docs/src/main/sphinx/release/release-477.md

```diff
@@ -0,0 +1,202 @@
+# Release 477 (dd MMM 2025)
+
+## General
```

Comment:
Entries should be sorted according to the following order:
* new features / improvements
* performance improvements
* bug fixes